.github/regression/appian.csv

```diff
@@ -0,0 +1,8 @@
+benchmark/appian_benchmarks/q01.benchmark
+benchmark/appian_benchmarks/q02.benchmark
+benchmark/appian_benchmarks/q03.benchmark
+benchmark/appian_benchmarks/q04.benchmark
+benchmark/appian_benchmarks/q05.benchmark
+benchmark/appian_benchmarks/q06.benchmark
+benchmark/appian_benchmarks/q07.benchmark
+benchmark/appian_benchmarks/q08.benchmark
```

Comment:
Does this need to be added to Regression.yml so CI runs it? If not, how does the performance compare with `main`?